src/main/java/frc/robot/subsystems/hood/Hood.java

```diff
@@ -3,14 +3,21 @@
 import edu.wpi.first.wpilibj.PneumaticsModuleType;
 import edu.wpi.first.wpilibj.Solenoid;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Robot;
+import frc.robot.Robot.*;
 
 import static frc.robot.Ports.Hood.SOLENOID;
 
 public class Hood extends SubsystemBase {
     private static Hood INSTANCE;
-    private final Solenoid angleChanger = new Solenoid(PneumaticsModuleType.CTREPCM, SOLENOID);
+    private final Solenoid angleChanger;
 
     private Hood() {
+        if (Robot.pneumaticsBase.checkSolenoidChannel(SOLENOID)) {
+            angleChanger = new Solenoid(PneumaticsModuleType.CTREPCM, SOLENOID);
+        } else {
+            angleChanger = null;
```

Comment:
Wouldn't that crash the code if i.e. `close()` is called?

Reply:
We wouldn't use the close it just simplifies the testing process if something is not connected yet

Reply:
Will the testing be added in another branch?